src/aten/UpSampleBilinear2d.cpp

```diff
@@ -0,0 +1,38 @@
+#include <ATen/Context.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <aten/sycl/UpSampleBilinear2dKernels.h>
+
+namespace at {
+Tensor& XPUNativeFunctions::upsample_bilinear2d_out(
+    const Tensor& self,
+    IntArrayRef output_size,
+    bool align_corners,
+    std::optional<double> scales_h,
+    std::optional<double> scales_w,
+    Tensor& output) {
```

Comment:
Lack of meta process.
<img width="947" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/33a40400-d8ca-4094-8939-e5ed1959e2cb">